src/frontend/admin/src/components/templates/organizations/form/translations.ts

```diff
@@ -1,6 +1,37 @@
 import { defineMessages } from "react-intl";
 
 export const organizationFormMessages = defineMessages({
+  generalSectionTitle: {
+    id: "components.templates.organizations.form.translations.generalSectionTitle",
+    defaultMessage: "General",
+    description: "Title for the general section",
```

Comment:
Maybe precise that it concerns organizations?

```suggestion
    description: "Title for the general section of organization form",
```